scripts/gulp/lint.js

```diff
@@ -50,15 +48,13 @@ gulp.task( 'lint:scripts', () => {
 /**
  * Lints the source LESS files for errors.
  */
-gulp.task( 'lint:styles', () => {
-  return gulp
-    .src( ['!src/cf-grid/src-generated/*.less', 'src/**/*.less'] )
-    .pipe( gulpStylelint( {
-      reporters: [
-        { formatter: 'string', console: true }
-      ]
-    } ) );
-} );
+gulp.task( 'lint:styles', () => gulp
```

Comment:
Curious why this one is ` => gulp ...` and not ` => { gulp ...` like the others?

Reply:
It's because it technically only contains one line of code. The brackets are for multiple lines. I'm unclear why ESLint fixed it this way and didn't for the other arrow functions that just contained one function call. I re-added the brackets and autofixed again and it didn't change it, so let's keep them for consistency.